app/views/swiss/form.scala

```diff
@@ -22,6 +22,13 @@ object form:
         div(cls := "swiss__form tour__form box box-pad")(
           h1(cls := "box__top")(trans.swiss.newSwiss()),
           postForm(cls := "form3", action := routes.Swiss.create(teamId))(
+            frag(
```

Comment:
`frag()` is used to group elements into one fragment, without adding a real parent element. it should not be needed here.